docs/registries/network.md

```diff
@@ -0,0 +1,297 @@
+---
+description: >-
+  The Network registry includes a range of utilities for working with network
+  resources, such as IP addresses, CIDR blocks, and network interfaces.
+---
+
+# Network
+
+{% hint style="info" %}
+You can easily import all the functions from the <mark style="color:yellow;">`network`</mark> registry by including the following import statement in your code
+
+```go
+import "github.com/go-sprout/sprout/registry/network"
+```
+{% endhint %}
+
+### <mark style="color:purple;">parseIP</mark>
+
+ParseIP parses a string representation of an IP address and returns its net.IP form. It attempts to parse the string as either an IPv4 or IPv6 address.
```

Comment:
Maybe something like this

```suggestion
ParseIP parses a string representation of an IP address and returns its `net.IP` form. It attempts to parse the string as either an IPv4 or IPv6 address.
```


```suggestion
ParseIP parses a string representation of an IP address and returns its "net.IP" form. It attempts to parse the string as either an IPv4 or IPv6 address.
```

Otherwise it looks like a strange syntax issue.